foia_hub/templates/includes/footer.html

```diff
@@ -2,10 +2,10 @@
 <footer>
   <div class="container">
       <a href="https://18f.gsa.gov">
-        <img src="/static/images/logo-18f.png" />
+        <img src="{{ static("images/logo-18f.png") }}" />
```

Comment:
We stop hardcoding the static asset URLs here, and start using this template convenience.